spotify-bin

```diff
@@ -2,7 +2,7 @@
 
 SCALE_FACTOR=`get-scale-factor.py`
 
-/app/extra/bin/spotify --force-device-scale-factor=$SCALE_FACTOR "$@" &
+env PULSE_PROP_application.icon_name="com.spotify.Client" /app/extra/bin/spotify --force-device-scale-factor=$SCALE_FACTOR "$@" &
```

Comment:
I think it would be cleaner to split this into two lines:
```
export PULSE_PROP_application.icon_name="com.spotify.Client"
/app/extra/bin/spotify --force-device-scale-factor=$SCALE_FACTOR "$@" &
```

Reply:
Ok if that does the trick it works for me. I'll update my PR.

Reply:
```export PULSE_PROP_application.icon_name="com.spotify.Client"
/app/extra/bin/spotify --force-device-scale-factor=$SCALE_FACTOR "$@" &```

Does not work